indico/htdocs/js/utils/forms.js

```diff
@@ -77,9 +102,33 @@
             });
         }).on('change input', function() {
             var $this = $(this);
-            var untouched = $this.serialize() == $this.data('initialData');
+            var untouched = $this.serialize() === $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
```

Comment:
What if there's another unrelated corner message? We should give that corner message a custom class to avoid selecting any other corner message.